testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/BruteForceTest.java

```diff
@@ -564,6 +565,7 @@ public void testNonExistingAccounts() throws Exception {
     }
 
     @Test
+    @Ignore // TODO: https://github.com/keycloak/keycloak/issues/20525
```

Comment:
How about @Ignore this test just with Chrome and Firefox, but still have it working with HtmlUnitDriver etc, since it fails just with chrome and FF?

I figured we have `@IgnoreBrowserDrivers` annotation for this.